torchgeo/datasets/nccm.py

```diff
@@ -55,12 +55,24 @@ class NCCM(RasterDataset):
 
     filename_regex = r"CDL(?P<year>\d{4})_clip"
     filename_glob = "CDL*.*"
-    zipfile_glob = "13090442.zip"
 
     date_format = "%Y"
     is_image = False
-    url = "https://figshare.com/ndownloader/articles/13090442/versions/1"
-    md5 = "eae952f1b346d7e649d027e8139a76f5"
+    urls = {
+        2017: "https://figshare.com/ndownloader/files/25070582",
```

Comment:
Can we sort these newest to oldest to match CDL? Maybe NLCD should be changed to. The idea is that newer versions will be added to the top, so you don't have to scroll as far to add them. But this is just the style used by Spack, we don't have to stick with it.